go.mod

```diff
@@ -11,129 +11,127 @@ require (
 	cloud.google.com/go v0.0.0-20160913182117-3b1ae45394a2
 	github.com/Azure/azure-sdk-for-go v21.3.0+incompatible
 	github.com/Azure/go-autorest v11.1.0+incompatible
-	github.com/GeertJohan/go.rice v0.0.0-20170420135705-c02ca9a983da // indirect
+	github.com/GeertJohan/go.rice v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
these versionless require directives aren't right... can you revert the `require` bits to this file (your new `replace` directives are fine) and rerun `hack/update-vendor.sh`?

Reply:
ok